src/partials/footer.html

```diff
@@ -1 +1 @@
-<footer></footer>
+<footer>TEST</footer>
```

Comment:
Co to za test? Usuwać tego branch'a?

Reply:
brancha nie usuwaj, to był test czy działa mi GH Desktop :)